src/instrumentsscene/qml/MuseScore/InstrumentsScene/internal/InstrumentsTreeItemDelegate.qml

```diff
@@ -442,6 +445,18 @@ FocusableControl {
                     horizontalCenter: undefined
                 }
             }
+        },
+
+        //! NOTE: Workaround for a bug in Qt 6.2.4 - see PR #24106 comment
```

Comment:
please add direct link to Qt Bug Reports and I think it's enough to write about this workaround here only